### PR TITLE
Feature/improve android export

### PR DESCRIPTION
**Pull request by victorlevasseur**
This displays a button to export to Android (by allowing Platforms to have multiple exporters) and let the user choose a directory. There's also a help button to redirect the user to the wiki (I'm currently writting the doc)

### CODE REVIEW

**Comment by victorlevasseur** [comment]
Don't merge it. I will add a better dialog to give more options to the user.

File: GDCpp/GDCpp/IDE/AndroidExporter.h [hunk 1]
```diff
@@ -9,7 +9,7 @@ class wxProgressDialog;
  * \brief Allow to export a game for Android
  * \todo Highly experimental
  *
- * To try it, replace the exporter in CppPlatform::GetProjectExporter and
+ * To try it, replace the exporter in CppPlatform::GetProjectExporters and
```
Comment (4ian) [inline GDCpp/GDCpp/IDE/AndroidExporter.h]: @victorlevasseur I think this comment should be rewritten as the exporter is now directly available from the File menu. Also maybe add a link to the wiki article being written? :)

**Comment by 4ian** [comment]
Looks good, I'm merging this! Maybe update the comment where I put a note though in another PR?

**Comment by 4ian** [comment]
Just saw your comment, not merging this for now! :wink:

**Comment by 4ian** [comment]
@victorlevasseur I'm thinking about this just now: what could be awesome with a dialog is to have a few options of course, but also automatic checks made by GDevelop to see if the Android SDK/NDK is installed on the system.
This could eventually be shared with the Cocos2d exporter. This would also ease the creation of exporters automatically calling the Android SDK/NDK in the future: the user would "just" have to install it and make sure GDevelop detects the SDKs (and python and cocos for Cocos2d).

**Comment by victorlevasseur** [comment]
Current wiki page : http://wiki.compilgames.net/doku.php/fr/gdevelop/documentation/manual/native_android_export (french only, works on both Windows and Linux)

**Comment by 4ian** [comment]
Looks quite complete (and a bit frightening for non developers ahah)!
I guess the translation to English would be good to get something that is complete so that some users can test the export and do some feedbacks :)

**Comment by victorlevasseur** [comment]
That should be ok to merge now.

**Comment by 4ian** [comment]
Sweet! :+1:

**Pull request merged**